pkg/distro/generic/images.go

```diff
@@ -579,34 +604,30 @@ func imageInstallerImage(workload workload.Workload,
 	img.Kickstart.Keyboard = img.OSCustomizations.Keyboard
 	img.Kickstart.Timezone = &img.OSCustomizations.Timezone
 
-	instCust, err := customizations.GetInstaller()
-	if err != nil {
-		return nil, err
-	}
-	if instCust != nil && instCust.Modules != nil {
-		img.EnabledAnacondaModules = append(img.EnabledAnacondaModules, instCust.Modules.Enable...)
-		img.DisabledAnacondaModules = append(img.DisabledAnacondaModules, instCust.Modules.Disable...)
-	}
-	img.EnabledAnacondaModules = append(img.EnabledAnacondaModules, anaconda.ModuleUsers)
-
 	img.Platform = t.platform
 	img.Workload = workload
 
 	img.ExtraBasePackages = packageSets[installerPkgsKey]
 
+	img.InstallerCustomizations, err = installerCustomizations(t, bp.Customizations)
+
+	if err != nil {
+		return nil, err
+	}
+
 	installerConfig, err := t.getDefaultInstallerConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	if installerConfig != nil {
-		img.EnabledAnacondaModules = append(img.EnabledAnacondaModules, installerConfig.EnabledAnacondaModules...)
-		img.AdditionalDracutModules = append(img.AdditionalDracutModules, installerConfig.AdditionalDracutModules...)
-		img.AdditionalDrivers = append(img.AdditionalDrivers, installerConfig.AdditionalDrivers...)
-		if img.Kickstart.Unattended {
-			img.AdditionalKernelOpts = append(img.AdditionalKernelOpts, installerConfig.KickstartUnattendedExtraKernelOpts...)
-		}
+	// TODO: left over bits
+	img.InstallerCustomizations.EnabledAnacondaModules = append(img.InstallerCustomizations.EnabledAnacondaModules, anaconda.ModuleUsers)
+
+	if img.Kickstart.Unattended {
+		img.InstallerCustomizations.AdditionalKernelOpts = append(img.InstallerCustomizations.AdditionalKernelOpts, installerConfig.KickstartUnattendedExtraKernelOpts...)
+	}
```

Comment:
What does the comment mean here (left over bits)?  A more clarifying TODO or an explanation of what's wrong or out of place would be nice here.

Reply:
Left over bits are things that I feel could move into the `installerCustomization` and `InstallerCustomizations` directly, maybe in the future (mostly `img.Kickstart` things). I'll adjust the comment.